README.md

```diff
@@ -11,19 +13,41 @@ For the Points developer take-home assignment, the following task is required: y
     * Calculates and displays the total taxes owed for the salary
     * Displays the amount of taxes owed per band
     * Displays the effective rate
+* There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
```

Comment:
Can you move this under the general instructions at the top please? This applies to all the assignments in theory.

Reply:
Done ✅